src/averageAges.js

```diff
@@ -1,61 +1,47 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function getAverage(arr) {
+  return arr
+    .map(user => user.died - user.born)
+    .reduce((acc, user) => acc + user) / arr.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let mans = people.filter(user => user.sex === 'm');
```

Comment:
- this comment is more grammatical, but still 😄 
```suggestion
  let men = people.filter(person => person.sex === 'm');
```